dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/ConfigHelper.java

```diff
@@ -0,0 +1,47 @@
+package io.dockstore.webservice.helpers;
+
+import java.util.Properties;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Helper class for setting the config variables for git commit info
+ */
+public final class ConfigHelper {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(ConfigHelper.class);
+
+    private ConfigHelper() {
+    }
+
+    /**
+     * Reads a properties file with git info and returns abbreviated commitId and build version.
+     * If file cannot be read, will log error and set config to "git property not found".
+     * @param fileName properties file with git information
+     * @return git commitID and buildVersion
+     */
+    public static GitInfo readGitProperties(final String fileName) {
+        final Properties properties = new Properties();
+        LOGGER.info("Retrieving git properties");
```

Comment:
should probably be `debug` (its a judgement call, but for me this message will always be shown and doesn't do anything aside from telling you where you are ... might even be `trace`)